src/EditorFeatures/CSharpTest/Diagnostics/Configuration/ConfigureSeverity/DotNetDiagnosticSeverityBasedSeverityConfigurationTests.cs

```diff
@@ -164,19 +164,7 @@ public async Task ConfigureEditorconfig_MaintainExistingEntry_None()
     </Project>
 </Workspace>";
 
```

Comment:
@jasonmalinowski Now that #36566 is merged in, `none` in editorconfig is correctly treated as a suppression by the compiler, so this test failure is expected. Not sure how merge from master into release\dev16.3-preview did not see these failures in CI.

Reply:
Oh unless this is another case where we had two CIs from the same source branch and it got misreported.

Reply:
Which merge PR was the one that broke it as best you can tell?

Reply:
![image](https://user-images.githubusercontent.com/10605811/60212915-402ef600-9817-11e9-8cd4-70b1d1ba3c49.png)

However, CI for #36765 shows it passed.

Reply:
Yeah, this is that bug again. Was going to wait until next week since I knew @sharwell had previously reached out but clearly this is more urgent.